### PR TITLE
Feat/add social media icons

### PR DESCRIPTION
**Pull request by ErikssonJoakim**
This PR adds the following icons:
- `github-round`
- `medium-round`
- `linkedin-round`
- `twitter-round`
- `telegram-round`
- `discord-round`

As well as a **breaking change** renaming the following icons:
- `facebook -> facebook-square`
- `linkedin -> linkedin-square`
- `twitter -> twitter-square`

<details>
<summary><b>Icon images</b></summary>

Capture d’écran 2022-10-18 à 15 58 02
Capture d’écran 2022-10-18 à 15 57 57

</details>

To not have redundant and useless information in the svgs this PR also adds the dev dependency `svgo`.

### CODE REVIEW

**Review by lolottetheclash — APPROVED** [review]
LGTM !

**Comment by fredericvilcot** [comment]
@ErikssonJoakim 
Please, do not link other repositories' issues in the description, event they are opensource.
This library must be fully agnostic about other projects, we can't take other issues from other projects as specifications.
Thanks 🙏

**Comment by fredericvilcot** [comment]
@ErikssonJoakim 
As this [commit](https://github.com/okp4/ui/pull/574/commits/da11361606a5dc4ad903167853240b110f72f052) is a breaking change, please reword it adding more context (this will appear in the changelog as a BC) please.

File: src/ui/atoms/icon/Icon.tsx [hunk 1]
```diff
@@ -18,8 +18,12 @@ export type IconName =
   | 'clock'
   | 'close'
   | 'cross'
-  | 'facebook'
-  | 'linkedin'
+  | 'discord-round'
+  | 'facebook-card'
```
Comment (fredericvilcot) [inline src/ui/atoms/icon/Icon.tsx:22]: I will have named it `facebook-square` to follow the `round` convention. 
Card does not mean a lot to me.

**Review by fredericvilcot — COMMENTED** [review]
Good, I added some comments about the naming, tell me 😌

**Comment by ErikssonJoakim** [comment]
@fredericvilcot Yes you are right the library needs to be agnostic I didn't think of that.

I will put the PR in draft until I've had the time to rename the BC commit and we've agreed on a suitable naming convention.

**Comment by fredericvilcot** [comment]
@ErikssonJoakim 
What's about this PR?

**Review by fredericvilcot — APPROVED** [review]
LGTM, thanks 😌

**Comment by ErikssonJoakim** [comment]
Thank you for your review @fredericvilcot. I've renamed the 'old' icons as requested and I believe the BC commit message now clearly states its purpose.

**Review by JoeyDjr — APPROVED** [review]

**Review by RaphaelEscrig — APPROVED** [review]

**Review by RaphaelEscrig — APPROVED** [review]

**Review by JoeyDjr — APPROVED** [review]

**Review by lolottetheclash — APPROVED** [review]
Even better now !

**Review by ccamel — APPROVED** [review]

**Review by ccamel — APPROVED** [review]

**Review by fredericvilcot — APPROVED** [review]

**Pull request merged**